Adding topic to update Log Forward CR (#26161)
openshift/openshift-docs: modules/cluster-logging-collector-log-forwarding-about.adoc

* `kafka`. A Kafka broker. The `kafka` output can use a TCP or TLS connection.

> * `default`. The internal {product-title} Elasticsearch instance. You do not need to configure the default output. If you do configure a `default` output, you receive an error message because the `default` output is reserved for the Cluster Logging Operator.

Reviewer: ```suggestion
* `default`. The internal {product-title} Elasticsearch instance. You must not configure the default output. If you do configure a `default` output, you receive an error message because the `default` output is reserved for the Cluster Logging Operator.
```

Author: I like the _You are not required to configure _ wording you suggested later
